src/datasets/utils/download_manager.py

```diff
@@ -184,6 +186,7 @@ def download(self, url_or_urls):
         )
         duration = datetime.now() - start_time
         logger.info("Downloading took {} min".format(duration.total_seconds() // 60))
+        self.downloaded_paths = downloaded_path_or_paths
```

Comment:
Sometimes users call `.download` several times.
This happens for example if you want to first download and read some kind of manifest that tells you which other files you need to download.

Maybe this could be a dict or a list that is updated each time `.download` is called ?

Reply:
Well seen! Thanks for pointing this out, @lhoestq: one, it can be called several times; two, it can return a list or a dict (or a tuple, or a ndarray,...). I am refactoring to take this into account.